requirements.devel.txt

```diff
@@ -53,6 +53,7 @@ SQLAlchemy-Continuum>=1.2.1 # dependency of invenio-db
 -e git+https://github.com/inveniosoftware/invenio-mail.git#egg=invenio-mail
 -e git+https://github.com/inveniosoftware/invenio-migrator.git#egg=invenio-migrator
 -e git+https://github.com/inveniosoftware/invenio-oaiserver.git#egg=invenio-oaiserver
+-e git+https://github.com/inveniosoftware/invenio-oauth2server.git#egg=invenio-oauth2server
```

Comment:
Can you please remove this line https://github.com/nikofil/cds/blob/54d4e3923faf595a29317d8e66627b3e3ca63c1e/requirements.devel.txt#L33